commands/info/index.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+
+const Command = require("@lerna/command");
+const output = require("@lerna/output");
+const envinfo = require("envinfo");
+
+module.exports = factory;
+
+function factory(argv) {
+  return new InfoCommand(argv);
+}
+
+class InfoCommand extends Command {
+  execute() {
+    output("\n Environment info:");
+    const info = await envinfo.run({
```

Comment:
Lerna 3.x still supports node v6.x, so we can't use `async`/`await` yet. Simple enough to refactor:
```js
return Promise.resolve()
  .then(() =>
    envinfo.run({
      // config
    })
  })
  .then(output);
```

Reply:
This isn't resolved, it's still using `await`